dvc/remote/base.py

```diff
@@ -804,7 +808,9 @@ def cache_exists(self, checksums, jobs=None, name=None):
 
         - Traverse: Get a list of all the files in the remote
             (traversing the cache directory) and compare it with
-            the given checksums.
+            the given checksums. Cache parent directories will
```

Comment:
should we update the whole comment that we have this autodection now? also, would be great to have a wiki or put a comment with some logic behind it - it's core and it's not very simple of others to understand

Reply:
updated the comment to explain how the autodetection works in 
65d7e22

If we keep `no_traverse` option (or rename/add a new one) this should also go into the dvc.org docs since `no_traverse` was not documented at all in the first place

Reply:
👍